### PR TITLE
Support Swift Package Manager

### PR DESCRIPTION
**Pull request by yutailang0119**

### CODE REVIEW

File: WebKitPlus.podspec [hunk 1]
```diff
@@ -15,8 +15,8 @@ Pod::Spec.new do |s|
     s.framework = "WebKit"
     s.source = { :git => "https://github.com/yashigani/WebKitPlus.git", :tag => "#{s.version}" }
 
-    s.source_files = "WebKitPlus/*.{swift,h}"
-    s.resource_bundles = { "WebKitPlus" => ["WebKitPlus/*.lproj"] }
+    s.source_files = "Sources/WebKitPlus/**/*.{swift,h}"
+    s.resources = "Sources/WebKitPlus/Sources/Resources/**/*.lproj"
```
Comment (ikesyo) [inline WebKitPlus.podspec:19]:
```suggestion
    s.resources = "Sources/WebKitPlus/Resources/**/*.lproj"
```

Redundant `Sources`.

Comment (ikesyo) [inline WebKitPlus.podspec:19]: Why `resource_bundles` is replaced with `resources`?

It seems that considering the case where it is built as a static library, `resource_bundles` is recommended.

- https://guides.cocoapods.org/syntax/podspec.html#resource_bundles
- https://guides.cocoapods.org/syntax/podspec.html#resources

Comment (yutailang0119) [reply WebKitPlus.podspec]: Right. I look good to use `resource_bundles` :+1:

**Review by ikesyo — APPROVED** [review]
👍

**Comment by yutailang0119** [comment]
Thx for reviewing!

**Pull request merged**